src/plugins/intel_cpu/src/mkldnn_graph_optimizer.cpp

```diff
@@ -451,6 +455,74 @@ void MKLDNNGraphOptimizer::FuseMultiplyAndAdd(MKLDNNGraph &graph) {
     }
 }
 
+void MKLDNNGraphOptimizer::MergeConvertAndScaleShift(MKLDNNGraph& graph) {
+    auto& graphNodes = graph.GetNodes();
+
+    auto isSuitableParentNode = [](MKLDNNNodePtr node) {
+        return node->getType() == Convert && node->getChildEdges().size() == 1 &&
+               (node->getOriginalInputPrecisionAtPort(0) == Precision::U8 ||
+                node->getOriginalInputPrecisionAtPort(0) == Precision::I8) &&
+               node->getOriginalOutputPrecisionAtPort(0) == Precision::FP32;
+    };
+
+    auto isSuitableChildNode = [](MKLDNNNodePtr node) {
+        return node->getType() == Eltwise && (node->getAlgorithm() == EltwiseMulAdd ||
```

Comment:
Why we limit supported algorithms? Eltwise node supports U8/I8 on input for any algorithm.